tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -332,6 +371,30 @@ def cancel_job(self, job_id):
         """Cancel a job"""
         return self._remote_command_executor.run_remote_command("scancel {}".format(job_id))
 
+    def set_nodes_to_down_state(self, nodes):
```

Comment:
nit: single function set_nodes_state?

also I think you can pass multiple nodes as NodeName so that you can run the remote command just once.